src/events/components/CalendarView/CalendarTile.tsx

```diff
@@ -20,15 +22,24 @@ export const createDayList = (amount: number, start: number): number[] => {
   return l;
 };
 
-export const CalendarEventTile = ({ events, active = true, day }: ITileProps) => {
+export const CalendarEventTile = ({ events, active = true, day, month }: ITileProps) => {
+  const thisDate = new Date();
+  const today = thisDate.getDate();
+  const thisMonth = thisDate.getMonth() + 1;
+  // tslint:disable-next-line: radix
+  const isToday = day === today && thisMonth === parseInt(month.toFormat('M')) ? true : false;

```

Comment:
I don't think this would take the year into account, so it could possibly turn a false positive if things line up correctly?